src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/LrdApiController.java

```diff
@@ -61,17 +77,83 @@
             message = "Internal Server Error"
         )
     })
-    @GetMapping(produces = APPLICATION_JSON_VALUE)
+    @GetMapping(
+        path = "/orgServices",
+        produces = APPLICATION_JSON_VALUE
+    )
     public ResponseEntity<Object> retrieveOrgServiceDetails(
         @RequestParam(value = "serviceCode", required = false) String serviceCode,
         @RequestParam(value = "ccdCaseType", required = false) String ccdCaseType,
         @RequestParam(value = "ccdServiceNames", required = false) String ccdServiceNames) {
         log.info("Inside retrieveOrgServiceDetails");
-
-        ConstraintValidation.validateInputParameters(serviceCode, ccdCaseType, ccdServiceNames);
+        ValidationUtils.validateInputParameters(serviceCode, ccdCaseType, ccdServiceNames);
```

Comment:
loggingComponentName missed here

Reply:
I guess this is out of the scope of the story. I know nothing about this endpoint. I have just renamed the class from `ConstraintValidation` to `ValidationUtils`.